### ISSUE TITLE
A typo in center_crop funtion?

### ISSUE THREAD

**Issue by lyuyunhuan** [google-research/simclr#155]
In the center_crop function in data_util.py:
Original code:
_crop_height, crop_width = _compute_crop_shape(
      image_height, image_width, **height / width**, crop_proportion)_

shouldn't the aspect_ratio be width / height instead of height / width? Like the following:
_crop_height, crop_width = _compute_crop_shape(
      image_height, image_width, **width / height**, crop_proportion)_

Thanks

**Comment by chentingpc** [comment]
Yes, you're right, this should be a typo. It shouldn't affect default behavior since we set both height and width to image_size, but it would be a bug if width and height were different. Thanks for pointing this out!
